src/utils/testing/mocks/mockView.ts

```diff
@@ -8,6 +8,8 @@ const view: ZetkinView = {
   id: 1,
   organization: {
     id: 1,
+    is_active: true,
+    sub_orgs: [],
```

Comment:
Back to my question (on the interface) about whether organizations always have these properties. They definitely don't in this case (when received as a property on a view, or some other object from the API), so here we are creating a mock that doesn't look like the data we're actually going to get from the API, and so we are potentially creating bugs that our tests won't catch because the tests are using a mock not based in reality.

There needs to be some sort of refactoring of the types for `ZetkinOrganization` here. To start, I think the changes introduced in this PR for recursive retrieval of organizations, should include a new type for that recursive structure, and not change the existing `ZetkinOrganization`.